platform/android/src/ch/opengis/qfield/QFieldProjectActivity.java

```diff
@@ -414,10 +482,131 @@ private boolean onItemLongClick(int position) {
     protected void onActivityResult(int requestCode, int resultCode,
                                     Intent data) {
         Log.d(TAG, "onActivityResult ");
+        Log.d(TAG, "requestCode: " + resultCode);
```

Comment:
This looks like a duplicate of below

Reply:
@m-kuhn , I had the wrong variable, fixed (resultCode -> requestCode)